src/CMakeLists.txt

```diff
@@ -598,30 +598,30 @@ endif()
 # --------------------
 # TODO: Install does not work as intended - needs work or maybe it should just be removed?
 install(TARGETS test_tool encrypt_file packagetool decrypt_file
-        DESTINATION bin
-        ARCHIVE DESTINATION lib)
+        DESTINATION ${CMAKE_INSTALL_BINDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
-install(TARGETS ${LVETARGET} DESTINATION bin/LVE)
+install(TARGETS ${LVETARGET} DESTINATION ${CMAKE_INSTALL_BINDIR}/LVE)
```

Comment:
Thank you for this PR this looks really useful 😄 .

One small comment:
Please also add `include(GNUInstallDirs)` at the top of this file.
I don't think the variable `${CMAKE_INSTALL_BINDIR}` is defined otherwise.